test/units/testsuite-55.sh

```diff
@@ -38,9 +38,20 @@ systemctl start testsuite-55-testchill.service
 systemctl start testsuite-55-testbloat.service
 
 # Verify systemd-oomd is monitoring the expected units
-oomctl | grep "/testsuite-55-workload.slice"
-oomctl | grep "20.00%"
-oomctl | grep "Default Memory Pressure Duration: 2s"
+# Try to avoid racing the oomctl output check by checking in a loop with a timeout
+oomctl_output=$(oomctl)
+timeout="$(date -ud "1 minutes" +%s)"
```

Comment:
Make it 5? Some of those CIs are _really_ congested

Reply:
Well, we went from a timeout of 0 s to 60 s here. Maybe this will be enough.

Reply:
ok, let's merge this and if needed bump it again